Presentation.Web/Extensions/IQueryableExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Web;
+
+namespace Presentation.Web.Extensions
+{
+    public static class IQueryableExtensions
```

Comment:
we don't need this one. It promotes not getting type checks on the lambda expressions we built (see earlier comment)